### DISCUSSION TITLE
FHSS MANET/Mesh voice?

### DISCUSSION THREAD

**Discussion by sgofferj** [jgromes/RadioLib#1092] [Q&A]
Is anyone aware of any open source project using radiolib for digital voice over FHSS? Maybe as a PTP/PTMP FHSS MANET? Maybe even as a FHSS MANET Mesh?

If not, which radios would you recommend to start tinkering with that kind of technology? (EU region)

**Comment by jgromes** [comment]
That seems to be a rather specific use-case; I'm not aware of any existing projects in this area.

SX127x supports FHSS (https://github.com/jgromes/RadioLib/blob/master/examples/SX127x/SX127x_Transmit_FHSS/SX127x_Transmit_FHSS.ino), while LR11x0 supports LR-FHSS (https://github.com/jgromes/RadioLib/tree/master/examples/LR11x0/LR11x0_LR_FHSS_Modem), not sure if that is useful to you.

**Reply by sgofferj** [reply]
Thanks! I'll get some 127x and play with them.
